packages/docs/docs/contracts-api/unirep-voting-sol.md

```diff
@@ -0,0 +1,102 @@
+---
+title: UniRep Voting contract
+---
+
+This is an example contract that utilizes the core UniRep contract and the Voting Prize NFT contract. `UnirepoVoting.sol` is initialized with the number of projects and `VotingPrizeNFT.sol`. 
+
+There are two types of users: hackers and voters. A hacker can join a project of a certain project ID. The voters and hackers can vote for a project ID with one upvote or one downvote. 
+
+The project ID has the highest number of vote counts is the winner. All hackers that are part of the project ID can claim an NFT prize.  
+
+To deploy the UnirepVoting contract, we need to first deploy `Unirep`, `EpochKeyVerifierHelper`,  `ReputationVerifierHelper`, `VotingPrizeNFT`. Then initialize the UnirepVoting contract with the deployed addresses and numTeams. 
+
+```typescript
+import {
+    UnirepVoting__factory,
+    VotingPrizeNFT__factory,
+} from '@unirep/contracts/typechain'
+
+  unirep = await deployUnirep(deployer)
```

Comment:
if it is not in the test
it should be `const unirep = await deployUnirep(deployer)`